tests/integration/README.md

```diff
@@ -0,0 +1,386 @@
+# you have to have the following output
+#  Id   Name                   State
+# ---------------------------------------
+#  -    c2r_centos7_template   shut off
+#  -    c2r_centos8_template   shut off
+#  -    c2r_oracle7_template   shut off
+#  -    c2r_oracle8_template   shut off
+
+# Each VM when turned on shou
+virsh qemu-agent-command --domain c2r_centos8_template '{"execute":"guest-ping"}'
+# {"return":{}}
+
+# possibility to ssh to each VM with your key (no password)
+ssh root@{VM_IP_ADDR} 
+```
+
+1. Install necessary dependencies
+
+```bash
+# install deps
+sudo dnf install -y \
+  make \
+  python3-virtualenv \
+  podman \
+  podman-docker \
+  ansible \
+  vagrant \
+  virt-install \
+  virt-manager \
+  virt-viewer \
+  qemu-kvm \
+  libvirt-devel \
+  libvirt-daemon-config-network \
+  libvirt-daemon-kvm \
+  libguestfs-tools
+
+# add libvirt collection
+ansible-galaxy collection install community.libvirt
+
+# add yourself to libvirt group, so might work with libvirt without sudo
+sudo usermod --append --groups libvirt $(whoami)
+
+# switch libvirt uri to a system to enable NAT support for networking
+echo 'uri_default = "qemu:///system"' >> ~/.config/libvirt/libvirt.conf
+
+# [OPTIONAL] Use custom image storage location (if you have limited space in /var/lib/libvirt/images)
+virsh pool-edit --pool default
+# adjust <path> to a new location could (could be any existing dir)
+#  <target>
+#    <path>/any/existing/dir</path>
+# say to qemu to use your user for qemu processes
+echo -e "user = \"$(whoami)\"\ngroup = \"$(whoami)\"" | sudo tee -a /etc/libvirt/qemu.conf
+
+# apply changes
+sudo systemctl restart libvirtd
+```
+
+2. Create and setup VMs
+```bash
+# cd to the c2r-vagrant repo dir, i.e.
+cd ~/Documents/c2r-vagrant
+
+# Build vagrant VM
+SYSTEM=centos8 SYSTEM_SESSION=1 vagrant up
+
+# [OPTIONAL] generate your ssh key if you still don't have one (ls -al ~/.ssh/*.pub)
+# follow instructions in https://docs.gitlab.com/ee/ssh/#generate-an-ssh-key-pair
+
+# copy you ssh key to the VM (you'll be asking to enter the password (`vagrant`)
+ssh-copy-id root@$(virsh domifaddr --domain c2r-vagrant_default | gawk 'match($0, /(192.+)\/.+/, ary) {print ary[1]}')
+
+# ssh to the vagrant VM
+ssh root@$(virsh domifaddr --domain c2r-vagrant_default | gawk 'match($0, /(192.+)\/.+/, ary) {print ary[1]}')
+    # update packages
+    yum update -y
+    
+    # Disable blacklisted rpc in qemu config
+    vim /etc/sysconfig/qemu-ga
+    # comment line starting with BLACKLIST_RPC and save (:wa)
+    #
+    
+    # Switch selinux into permissive mode
+    vim /etc/selinux/config
+    # set SELINUX=permissive as save (:wa)
+    #
+    
+    # enable qemu-ga
+    systemctl enable qemu-guest-agent
+    
+    # install python pip
+    # for centos8 and ol8
+    curl https://bootstrap.pypa.io/pip/get-pip.py | python3
+    # for centos7 and ol7
+    # curl https://bootstrap.pypa.io/pip/2.7/get-pip.py | python
```

Comment:
Installing pip is part of the basic_setup.yml: `name: Install pip on Centos 7`
Isn't it duplication of steps?

Reply:
Agree, removed from the playbook, as it has more sense to have it in the basic setup